### ISSUE TITLE
sslstrip: drop / upgrade to py3 / replace

### ISSUE THREAD

**Issue by gkroon** [pentoo/pentoo-overlay#616]
This nice tool is aging, no longer maintained and depends on py2. Shall we just remove it?

**Comment by blshkv** [comment]
this is an old software. Should be possible to replace with bettercap

**Comment by gkroon** [comment]
I don't think this is right, there is no v0.9.2. There's a v0.9-2, though, but that one cannot be built with Python 3. Can't I just remove it?

**Comment by gkroon** [comment]
Let me rephrase. Did you manage to build this successfully?

**Comment by blshkv** [comment]
No, even setup.py is not python3 compatible
